packages/snaps-types/src/types.d.ts

```diff
@@ -22,7 +22,9 @@ export type OnCronjobHandler = (args: {
   request: JsonRpcRequest<unknown[] | { [key: string]: unknown }>;
 }) => Promise<unknown>;
 
-export type SnapProvider = MetaMaskInpageProvider;
+export type SnapAPI = { request: StreamProvider['request'] };
```

Comment:
Maybe "SnapsProvider"?

Reply:
afaik we specifically wanted to move away from calling it a provider

Reply:
"SnapsAPI"?

Reply:
Maybe? 🤔

Reply:
@rekmarks

Reply:
`SnapsGlobalObject`, perhaps? I'm inspired by the TC39 spec, where e.g. the `JSON` global is just called "The JSON object". In natural language we should refer to the `SnapsGlobalObject` as "the `snaps` object" or "the global `snaps` object".

Reply:
Wait, this seems to imply that the object should be called `snaps`. Is that intended?